vmm/src/lib.rs

```diff
@@ -1972,57 +1951,44 @@ impl Vmm {
             }
         };
 
-        match request {
-            VmmAction::ConfigureBootSource(boot_source_body, sender) => {
-                Vmm::send_response(
-                    self.configure_boot_source(
-                        boot_source_body.kernel_image_path,
-                        boot_source_body.boot_args,
-                    ),
-                    sender,
-                );
-            }
-            VmmAction::ConfigureLogger(logger_description, sender) => {
-                Vmm::send_response(self.init_logger(logger_description), sender);
-            }
-            VmmAction::FlushMetrics(sender) => {
-                Vmm::send_response(self.flush_metrics(), sender);
-            }
-            VmmAction::GetVmConfiguration(sender) => {
-                Vmm::send_response(
-                    Ok(VmmData::MachineConfiguration(self.vm_config.clone())),
-                    sender,
-                );
-            }
-            VmmAction::InsertBlockDevice(block_device_config, sender) => {
-                Vmm::send_response(self.insert_block_device(block_device_config), sender);
+        let (response, sender) = match request {
```

Comment:
nit: I prefer it how it was before.

Reply:
Okay, I was trying with this change to remove the repetitious Vmm::send_response / Vmm::Action part, but this can be reverted -- although there might be a compromise ?

Reply:
Probably this code will be massively refactored any way when we are going to switch to a custom HTTP server. I wouldn't invest too much in it.

Reply:
Ok, sure, I'll revert this change.